### PR TITLE
Contract page

### PR DESCRIPTION
**Pull request by ca1yps0**

### CODE REVIEW

File: app/styles/application.scss [hunk 1]
```diff
@@ -14,14 +14,23 @@ body > .ember-view
...
   .el-label-dark{ @include label; color: $color-text-dark;}
-
+/*
+$fs6 : 2.3rem;
+$fs5 : 1.5rem;
+$fs4 : 1.3rem;
+$fs3 : 1.1rem;
+$fs2 : 1rem;
+$fs1 : 0.8rem;
+$fs0 : 0.6rem;
```
Comment (nightsh) [inline app/styles/application.scss]: Not really happy with `fsX` names. I have no idea what it means :stuck_out_tongue_winking_eye:

File: app/styles/network/show.scss [hunk 2]
```diff
@@ -28,7 +28,7 @@ div#network-container {
   .ember-view { height: 100%; }
 
   div#details-container > * {
-    margin: 0; padding: 0; border: 0;
+    margin: 0; padding: 0; /*border: 0;*/
```
Comment (nightsh) [inline app/styles/network/show.scss]: Don't comment, just remove it. I'm sure we know how to put it back if we need it.

File: app/styles/network/show/details.scss [hunk 3]
```diff
@@ -1,7 +1,7 @@
 div#details-sidebar {
   @include bg-dark;
   z-index: 9;
-  border-right: 1px solid #292929;
+ /* border-right: 1px solid #292929;*/
```
Comment (nightsh) [inline app/styles/network/show/details.scss]: Remove it if we don't need it.

File: app/styles/network/show/details.scss [hunk 4]
```diff
@@ -11,7 +11,9 @@ div#details-sidebar {
   clear: both;
 
   .details{
-    margin-top: 65px;
+    margin-top: 40px;
+    margin-left: 10px;
+    i{cursor: pointer;}
```
Comment (nightsh) [inline app/styles/network/show/details.scss]: Adjust formatting here.

File: app/styles/network/show/details.scss [hunk 5]
```diff
@@ -22,9 +24,10 @@ div#details-sidebar {
 
   /*change network name*/
   div.network-input {
+    margin-bottom: 0px;
     margin-top: 20px;
-    input { height:  50px !important;}
-    .controls { line-height: 60px;}
+    input { height:  35px !important; margin-bottom: 0px}
```
Comment (nightsh) [inline app/styles/network/show/details.scss]: Leave an empty space before closing brace `}`.
And you forgot the semicolon at the end of the line.

File: app/styles/network/show/details.scss [hunk 6]
```diff
@@ -22,9 +24,10 @@ div#details-sidebar {
...
+    .controls { line-height: 44px;}
```
Comment (nightsh) [inline app/styles/network/show/details.scss]: Space before closing brace `}`

File: app/styles/network/show/details/contract.scss [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+    @include bg-dark;
+
+    .row1 {
+      padding-top: 40px;
+      hr{
+        margin-top: 20px;
+        margin-bottom: 30px;
+      }
+    }
+    .row2{
```
Comment (nightsh) [inline app/styles/network/show/details/contract.scss]: Space before opening brace `{`

File: app/styles/network/show/details/show.scss [hunk 8]
```diff
@@ -22,13 +22,23 @@ div#details-info {
...
       }
     }
-    .s3 {
-      width: 30%;
-    }
+
     .container-entity {
       width: 80%;
+
+      .second-row{
```
Comment (nightsh) [inline app/styles/network/show/details/show.scss]: Space before opening brace `{`

File: app/styles/network/show/details/show.scss [hunk 9]
```diff
@@ -22,13 +22,23 @@ div#details-info {
...
+        .s2{width: 19%;}
```
Comment (nightsh) [inline app/styles/network/show/details/show.scss]: More formatting.

File: app/templates/network/show/details/contract.hbs [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+              </div>
+            </div>
+            <div class="other-supplier">
+              <div class="col s8">
+                {{fa-icon "circle" size="1" class="fs3 supplier-color"}}
+                <span class="dark-text fs3">Supplier Name</span>
+              </div>
+              <div class="col s4 right-align el-title ">
+                <span class="fs5">12.4</span>
+                <span class="fs3">M</span>
```
Comment (nightsh) [inline app/templates/network/show/details/contract.hbs]: No need for this anymore. We'll have a helper in soon.

File: app/routes/network/show/details/contract.js [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+          'X-User-Token': `${token}`
+        }
+      }).then(
+        (data) => {
+          let results = data.search.results[0];
+          results.supplier = results.suppliers[0];
+          results.entityId = entityId;
+
+          let details = {"yes" : "check", "no" : "close", "null" : "question"};
+          let dateHash = {
```
Comment (ca1yps0) [inline app/routes/network/show/details/contract.js]: done

File: app/routes/network/show/details/contract.js [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+            "12": "Dec"
+          };
+
+          results.award.date.x_month = dateHash[results.award.date.x_month];
+          results.x_eu_project = String(results.x_eu_project);
+          results.x_framework = String(results.x_framework);
+          results.x_subcontracted = String(results.x_subcontracted);
+
+
+          results.eu_project = details[results.x_project] ? details[results.x_project] : "check" ;
```
Comment (ca1yps0) [inline app/routes/network/show/details/contract.js]: done

File: app/routes/network/show/details/contract.js [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+
+          return results;
+        }, (response) => {
+
+          self.get('notifications').clearAll();
+          _.forEach(response.errors, (error, index) => {
+            self.get('notifications').error(`Error: ${index } ${error.title}`);
+          });
+        });
+  },
```
Comment (ca1yps0) [inline app/routes/network/show/details/contract.js]: done

File: app/styles/network/show.scss (hunk 2 above)
Comment (ca1yps0) [inline app/styles/network/show.scss]: done

File: app/styles/network/show/details.scss (hunk 3 above)
Comment (ca1yps0) [inline app/styles/network/show/details.scss]: done

File: app/styles/network/show/details.scss (hunk 5 above)
Comment (ca1yps0) [inline app/styles/network/show/details.scss]: done

File: app/styles/network/show/details.scss (hunk 6 above)
Comment (ca1yps0) [inline app/styles/network/show/details.scss]: done

**Pull request merged**